instrumentation/active_record/lib/opentelemetry/instrumentation/active_record/patches/querying.rb

```diff
@@ -18,9 +18,17 @@ class << base
 
           # Contains ActiveRecord::Querying to be patched
           module ClassMethods
-            def find_by_sql(sql, binds = [], preparable: nil, &block)
-              tracer.in_span("#{self}.find_by_sql") do
-                super
+            if Gem::Version.new(Rails::VERSION::STRING) >= Gem::Version.new("7.2.0.alpha")
+              def find_by_sql(sql, binds = [], preparable: nil, allow_retry: false, &block)
+                tracer.in_span("#{self}.find_by_sql") do
+                  super
+                end
+              end
```

Comment:
On second thought we could capture all args and forward them instead of declaring them since they are not in use right?
```suggestion
              def find_by_sql(...)
                tracer.in_span("#{self}.find_by_sql") do
                  super
                end
              end
```

Reply:
Done